app-android/src/main/java/com/ustadmobile/port/android/view/CourseTerminologyEditFragment.kt

```diff
@@ -93,4 +104,81 @@ class CourseTerminologyEditFragment: UstadEditFragment<CourseTerminology>(), Cou
             terminologyEntryAdapter?.submitList(value)
             terminologyEntryAdapter?.notifyDataSetChanged()
         }
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+private fun CourseTerminologyEditScreen(
+    uiState: CourseTerminologyEditUiState = CourseTerminologyEditUiState(),
+    onTerminologyTermChanged: (TerminologyEntry?) -> Unit = {},
+    onCtTitleChanged: (String?) -> Unit = {}
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+    )  {
+
+        item{
+            ListItem(
```

Comment:
Because we are using textfields, we should not put them inside a ListItem. They are not really normal ListItems (e.g. they don't have a title, subheader, icon, etc).